### ISSUE TITLE
Adding Mowzies Mobs Creates INSANE lag on singleplayer

### ISSUE THREAD

**Issue by celsiusqc** [BobMowzie/MowziesMobs#340]
https://gist.github.com/celsiusqc/5b724cf61a94a82e6436d1851e36eae4

**Comment by celsiusqc** [comment]
Removing the mod fixes the issue, so hopefully this can be fixed

**Comment by BobMowzie** [comment]
Is the lag only when you are in the end?

**Comment by celsiusqc** [comment]
> Is the lag only when you are in the end?

No, it was happening in the overworld severely. Even more when exploring new chunks

**Comment by BobMowzie** [comment]
If you remove all mods except Mowzie’s Mobs, does it go away?

**Comment by QuerrixTU** [comment]
With only mowzie's mobs the lag is just reduced but still present.

**Comment by BobMowzie** [comment]
Alright. Strange...

**Comment by bryan161** [comment]
Lol, I got a species of lag. Its like no fps down but all is slow. Like open chest, or crafting. And its just when I add the mod

**Comment by cursinorock** [comment]
same here

**Comment by EmperorPlebpatine** [comment]
Is the lag present in multiplayer?

**Comment by jangro** [comment]
If a "me too" is helpful, same happening to me. no crashes, just very laggy and constant "base value out of range" errors in the console. It's a multiplayer server running Valhelsia 3 and this is the only mod I've added.

I removed only this mod and it all cleared up.

**Comment by BobMowzie** [comment]
Ooh wait send me a snippet of those base value out of range console errors, please!

**Comment by seymourimadeit** [comment]
`[26Mar2021 19:41:07.713] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:08.000] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:08.336] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:08.603] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:08.846] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:09.107] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:09.373] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:09.654] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
[26Mar2021 19:41:10.149] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]`

This is spammed in the log while in the end dimension.

**Comment by BobMowzie** [comment]
No idea what's wrong. I simply can't reproduce this issue. I guess I'll just leave it open for now; maybe another one of my fixes already resolved it.

**Comment by QuerrixTU** [comment]
> No idea what's wrong. I simply can't reproduce this issue. I guess I'll just leave it open for now; maybe another one of my fixes already resolved it.

I personaly use Java 8 OpenJ9 By AdoptJdk idk if the issue is from that need to test but it's late where i live lol.

**Comment by QuerrixTU** [comment]
I tested it and the issue is not due to the java used..

**Comment by Suraru** [comment]
Am having same issue, unable to load world, do not have betterend installed. 
[debug.log](https://github.com/BobMowzie/MowziesMobs/files/6225496/debug.log)
[latest.log](https://github.com/BobMowzie/MowziesMobs/files/6225497/latest.log)

Removing Mobs fixes the issue. Not sure what caused it, this is the first time I've seen this error. An important pattern is 
`[29Mar2021 20:55:53.199] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Base value out of range: 256 [-10-128]
`
will appear before
`[29Mar2021 20:55:53.199] [Server thread/ERROR] [net.minecraft.util.registry.WorldSettingsImport/]: Error adding element: Value 128 outside of range [0:64]
`
is spammed

**Comment by NovaN4** [comment]
I just installed the recent update (30 mins ago) of Mowzie's Mobs, went into my game and I can't find the lag I used to get on the first 1.16 version. I think you fixed it, but I can't be sure; it's not a very reliable bug and tends to be hard to reproduce for me (old jar didn't give me the issue until specifically teleporting into a jungle on one world, but on another world my game dropped to 0 tps simply upon entering the end)

**Comment by QuerrixTU** [comment]
Yes the lag is fixed in the last update, thank's Bob !
